src/types.ts

```diff
@@ -446,6 +454,8 @@ export interface RevolutPayOptions extends CommonOptions {
   email?: string
   /** Styles of the RevolutPay button */
   buttonStyle?: ButtonStyleOptions
+  /** Prefiiled person details of user. Only valid details are prefilled. */
+  prefilledPersonalDetails?: PrefilledPersonalDetails
```

Comment:
can we please rename this to `customer` ? It'll be inline with other widget method e.g., card popup.

Reply:
e.g., as close to the existing `CustomerDetails` type here (line 367) as possible